controllers/reviewController.js

```diff
@@ -0,0 +1,16 @@
+const reviewService = require("../services/reviewService");
+const ErrorCreater = require("../middlewares/errorCreater");
+
+const getReviews = async (req, res) => {
```

Comment:
```suggestion
const getReviewsOfPin = async (req, res) => {
```
네이밍 정확하게 해주세요. getReviews는 말그대로 모든 리뷰를 전부 불러오는 겁니다. 이름에서 그게 어떤 값으로 필터링되는것인지 알 수 없으면 나중에 다른 사람이 본인 상황에 해당하는 api가 개발이 된 적이 없는 것 같아 새롭게 getReviewsOfPin 을 다시 개발할 수도 있습니다. 

전체적으로 다시 한 번 살펴보시고 모든 PR에 적용해 주세요. 다른 PR들도 전부 점검하고 다시 라벨 변경해 주세요.

Reply:
dao단의 경우에는 getReviewsListOfPin 이라고 할때 뭔가 이름이 너무 길어져서 일단은 getReviewList 라고 네이밍 했습니다.

Reply:
이름이 길어진다고 해서 가독성이 떨어지면 더 큰 문제입니다.. 게다가 그렇게 길지도 않구요.